scripts/person/person.gd

```diff
@@ -833,7 +844,8 @@ func calculateprice():
 	var bonus = 1
 	price = beautybase*variables.priceperbasebeauty + beautytemp*variables.priceperbonusbeauty
 	price += (level-1)*variables.priceperlevel
-	price = price*globals.races[race.replace('Halfkin', 'Beastkin')].pricemod
```

Comment:
Note to self, take a look at pricemod, determine whats going on here. I might have overwritten something here.

Reply:
this code is consistent with Ralph's mod, though it is likely a balance change without a condition